nomad/eval_broker.go

```diff
@@ -28,6 +28,14 @@ var (
 
 	// ErrNackTimeoutReached is returned if an expired evaluation is reset
 	ErrNackTimeoutReached = errors.New("evaluation nack timeout reached")
+
+	// initialNackReenqueueDelay is the delay applied before re-enqueuing a
+	// Nacked evaluation for the first time
+	initialNackReenqueueDelay = time.Second
```

Comment:
Let's thread these through a config like `EvalNackTimeout`, that way there isn't a weird hack for testing either.